java/src/main/java/com/marginallyclever/makelangelo/MachineConfiguration.java

```diff
@@ -504,12 +512,12 @@ public void SaveConfig() {
 	/** 		TODO finish these methods.
 
 	 public boolean GetCanUseCloud() {
-		return prefs.getBoolean("can_use_cloud", false);
+		return machinePreferences.getBoolean("can_use_cloud", false);
 	}
 
 	
 	public void SetCanUseCloud(boolean b) {
-		prefs.putBoolean("can_use_cloud", b);
+		machinePreferences.putBoolean("can_use_cloud", b);
```

Comment:
copy pasta refactor error. should be makelangeloPreferences. Saving this comment to deal with it later as for now cloud logic isn't actually wired up.